packages/medusa/src/commands/seed.ts

```diff
@@ -58,33 +58,30 @@ const seed = async function ({ directory, migrate, seedFile }: SeedOptions) {
 
   const featureFlagRouter = featureFlagLoader(configModule)
 
-  const dbType = configModule.projectConfig.database_type
-  if (migrate && dbType !== "sqlite") {
```

Comment:
Question: Should we keep the `if(migrate)` to only run migrations if flag is passed along as `true`?

Reply:
Nice catch - will add back 👍